index.bs

```diff
@@ -46,36 +51,41 @@ Transport {#transport}
 Message transport is provided using a [[RFC6455|WebSocket]]
 connection.
 
-A [=remote end=] has a <dfn>connection</dfn>, which is a network connection, and
-a <dfn>websocket url</dfn>, which is a string. These are both initially null.
+A WebDriver [=session=] has a <dfn>WebSocket connection</dfn>, which
+is a network connection, and a <dfn>WebSocket URL</dfn>, which is a
+string. These are both initially null.
 
-When data is recieved on [=connection=] it must be processed according to the
+When data is recieved on [=WebSocket connection=] it must be processed according to the
 requirements of the [[!RFC6455|WebSockets]] specification. When [=a WebSocket
 message has been received=], an implementation must [=handle an incoming
 message=]. When [=the WebSocket closing handshake is started=] an implementation
 must [=end the session=].
 
-When required to <dfn>ensure a connection is established</dfn>, run
-the following steps:
+When required to <dfn>Establish a WebSocket Connection</dfn>, with
+argument <var>session id</var>, run the following steps:
```

Comment:
Can you pass in *session* rather than *session id* here? This way we can say "session's WebSocket URL" below, which is better.